package.json

```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/indiemoney/react-native-charts-wrapper"
   },
-  "version": "0.2.11",
+  "version": "0.2.12",
```

Comment:
been doing it but the author isn't very responsible: https://github.com/wuxudong/react-native-charts-wrapper/pulls